app/dates.js

```diff
@@ -0,0 +1,15 @@
+const moment = require('moment')
+
+exports.todayYMD = () => {
+  return moment().format('YYYY-MM-DD')
+}
+
+exports.todayWOY = () => {
+  return moment().isoWeek()
+}
+
+// Months a returned zero based from moment
+// We add 1 to make sure January does not fail a truth test
+exports.todayMonth = () => {
+  return moment().month() + 1
```

Comment:
I hate to suggest this since you already tested, but moment adds another 1MB to the package. Is it worth it for this? I think this is possible with plain JS.